backend/src/graphql_api/account.rs

```diff
@@ -1291,6 +1291,94 @@ impl Account {
             account_index: self.index,
         }
     }
+
+    async fn plts(
+        &self,
+        ctx: &Context<'_>,
+        #[graphql(desc = "Returns the first _n_ elements from the list.")] first: Option<u64>,
+        #[graphql(desc = "Returns the elements in the list that come after the specified cursor.")]
+        after: Option<String>,
+        #[graphql(desc = "Returns the last _n_ elements from the list.")] last: Option<u64>,
+        #[graphql(
+            desc = "Returns the elements in the list that come before the specified cursor."
+        )]
+        before: Option<String>,
+    ) -> ApiResult<connection::Connection<String, AccountProtocolToken>> {
+        let pool = get_pool(ctx)?;
+        let config = get_config(ctx)?;
+
+        let query = ConnectionQuery::<DescendingI64>::new(
+            first,
+            after,
+            last,
+            before,
+            config.plt_by_account_address_connection_limit,
+        )?;
+
+        let mut row_stream = sqlx::query_as!(
+            AccountProtocolToken,
+            "
+            SELECT * FROM (
+                SELECT plt_tokens.name as token_name, plt_tokens.token_id, pa.amount as amount, plt_tokens.decimal as decimal, pa.token_index
+                    FROM public.plt_accounts AS pa
+                    INNER JOIN plt_tokens 
+                        ON pa.token_index = plt_tokens.index
+                    WHERE account_index = $1 
+                        AND amount > 0 
+                        AND $3 < pa.token_index AND pa.token_index < $2
```

Comment:
So far it seems we don't have an index to support this query.

Reply:
Created a Index for this query Please let me know